native/dgeqrf.go

```diff
@@ -43,20 +43,21 @@ func (impl Implementation) Dgeqrf(m, n int, a []float64, lda int, tau, work []fl
 	if k == 0 {
 		return
 	}
-	nbmin := 2 // Minimal number of blocks
+	nbmin := 2 // Minimal block size.
 	var nx int // Use unblocked (unless changed in the next for loop)
 	iws := n
 	ldwork := nb
-	// Only consider blocked if the suggested number of blocks is > 1 and the
-	// number of columns is sufficiently large.
-	if nb > 1 && k > nb {
-		// nx is the crossover point. Above this value the blocked routine should be used.
+	// Only consider blocked if the suggested block size is > 1 and the
+	// number of rows or columns is sufficiently large.
+	if 1 < nb && nb < k {
```

Comment:
I think this order is clearer but I can drop it if considered unnecessary. My main goal was to make the comments accurate.